api-server/server/boot/user.js

```diff
@@ -202,10 +202,15 @@ function createPostReportUserProfile(app) {
     log(username);
     log(report);
 
-    if (!username || !report || report === '') {
+    if (!username) {
       return res.json({
         type: 'danger',
-        message: 'Oops, something is not right please re-check your submission.'
+        message: 'Please enter a valid username.'
+      });
+    } else if (!report || report === '') {
```

Comment:
This conditional is not valid. What happens when `username`, `report` both are null/undefined?